src/backend/joanie/core/models/certifications.py

```diff
@@ -42,14 +40,8 @@ class Certificate(models.Model):
     Certificate represents and records all user certificates issued as part of an order
     """
 
-    certificate_definition = models.ForeignKey(
-        CertificateDefinition,
-        verbose_name=_("certificate definition"),
-        related_name="certificates_issued",
-        on_delete=models.RESTRICT,
-    )
     order = models.OneToOneField(
-        products_models.Order,
+        "core.Order",  # pylint: disable=all
```

Comment:
If someone has a better idea?
Astroid is looking for a module call models :/ 
but we only have a folder structure https://github.com/openfun/joanie/tree/main/src/backend/joanie/core/models
https://app.circleci.com/pipelines/github/openfun/joanie/281/workflows/8eb36ba7-d877-4198-b143-b50d3765c067/jobs/2124/parallel-runs/0/steps/0-106

Reply:
Just add a comment to explain why you have to disable pylint for this line.

Reply:
Maybe mention this [issue](https://github.com/PyCQA/pylint-django/issues/243) to help our readers to understand the problem and to help us to track the progress of this issue.